README.md

```diff
@@ -53,6 +53,19 @@ response = connection.post('https://httpbingo.org/post', { a: 1, b: 'foo', c: tr
 puts response.body['data'] ## => <a>1</a><b>foo</b><c>true</c>
 ```
 
+### Gyoku options
+
+The middleware uses [`gyoku` gem](https://rubygems.org/gems/gyoku) for XML encoding.
+
+You can pass options for it:
+
+```ruby
+connection = Faraday.new do |faraday|
+  ## https://www.rubydoc.info/gems/gyoku/#array-values
+  faraday.request :xml, gyoku_options: { unwrap: true }
```

Comment:
I am not sure about calling it `gyoku_options`, it seems too precise.

If the parser changes, it means this has to change too, and I don't think it should.

WDYT of something more generic like `parser_options` or in that style?

Reply:
I am just thinking of somebody using the gem in their code, and would have to call a `gyoku_options` without knowing what's gyoku 🤔 
That feels a bit weird to me, but I see your point about explicitness, and purposefully making a breaking change if the xml parser is switched. 

Having a general name for now could work, and if something else is introduced, then the proxy would be needed to pass things along. 

I am ok with both ways really, so feel free to pick your fav :)

Reply:
> I am just thinking of somebody using the gem in their code, and would have to call a `gyoku_options` without knowing what's gyoku 🤔

I'm not going to duplicate a documentation about parser options, so if anyone want to use them, modify parser behavior — it's a good reason and tip where they should dig.